qualtran/bloqs/chemistry/thc/prepare_test.py

```diff
@@ -36,12 +39,32 @@ def test_thc_prepare(bloq_autotester):
     bloq_autotester(_thc_prep)
 
 
+def build_random_test_integrals(num_mu: int, num_spat: int, seed: Optional[int] = None):
+    """Build random THC integrals for testing / demonstration purposes.
+
+    Args:
+        num_mu: The THC auxiliary dimension.
+        num_spat: The number of spatial orbitals.
+        seed: If not None then seed the rng with this value. Otherwise seed is
+            not set explcitly in this function.
+
+    Retuns:
+        t_l: The eigenvalues of the one-body Hamiltonian.
+        zeta: The central tensor (matrix) of size num_mu x num_mu.
+    """
+    if seed is not None:
+        np.random.seed(seed)
```

Comment:
can you make a new np.random.RandomState instead of (re-)seeding the global generator? And the constructor will do the correct thing for `None` values so you don't need the conditional

Reply:
TIL